WordPress/Classes/Services/WordPressComOAuthClientFacade.h

```diff
@@ -0,0 +1,22 @@
+#import <Foundation/Foundation.h>
+
+@protocol WordPressComOAuthClientFacade
```

Comment:
The methods in this class feel like something that should fall into the domain of our API classes.